lib/systems/examples.nix

```diff
@@ -109,6 +109,10 @@ rec {
     platform = platforms.riscv-multiplatform "32";
   };
 
+  mmix = {
+    config = "mmix";
```

Comment:
I think I would rather that this be
```suggestion
    config = "mmix-none";
```
which is a bit more of a normal form?

Reply:
I think I tried this and GCC couldn't build. I'll try and see.

Reply:
Actually, How does the GNU hello example work? Is this using glibc? In that case we should call it:
```suggestion
    config = "mmix-unknown-linux-gnu";
```

Reply:
Oh I see https://github.com/bminor/newlib/blob/master/newlib/libc/sys/mmixware/write.c ; so the emulator has some very basic syscalls. The syscall interface is what the OS triple comes from (it's the interface not implementation that counts).

I'll keep on researching.

Reply:
OK `mmix-knuth-mmixware` is a thing that goes up, where `mmixware` refers to that syscall interface. Can you try `mmix-unknown-mmixware`? (The vendor field is pretty stupid so I rather stick with `unknown` if we can.)

Reply:
```suggestion
    config = "mmix-unknown-mmixware";
```

Reply:
Ok, trying.

Reply:
error: Target specification with 3 components is ambiguous
(use '--show-trace' to show detailed location information)

Reply:
I see, I need to adjust `parse.nix` as needed.

Reply:
And note the special case should be `mmixware` not `mmix`.

Reply:
Done.